src/main.ts

```diff
@@ -524,6 +525,14 @@ function visitServices(
   }
 }
 
+function generateObjectInitStatement(func: FunctionSpec, fullName: string, nullPrototypes: boolean) {
+  if (nullPrototypes) {
+    return func.addStatement('const message = {...base%L} as %L', fullName, fullName);
+  } else {
+    return func.addStatement('const message = Object.create(base%L) as %L', fullName, fullName);
```

Comment:
nit: `else` after `return` is unnecessary noise

Reply:
Fixed